stats/opentelemetry/experimental/grpc_trace_bin_propagator.go

```diff
@@ -16,7 +16,7 @@
  *
  */
 
-package opentelemetry
+package experimental
```

Comment:
we should put experimental notice for `GRPCTraceBinPropagator` as well?

Reply:
Done